variables.tf

```diff
@@ -34,4 +34,11 @@ variable "dashboard_configs_folder" {
 variable "grafana_url" {
   description = "The url to your Grafana instance."
   type        = string
+}
+
+variable "asm_api_token_name" {
```

Comment:
this should be token value?

Reply:
It should be the name where the data source can pull the token from ASM